app/views/posts/posts.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Post </h1>
```

Comment:
- [ ] You have the post show view, but you are missing the posts index page, `app/views/posts/index.html.erb`